capture.cpp

```diff
@@ -979,7 +977,7 @@ const char *locale = DEFAULT_LOCALE;
         for ( ; i < argc - 1 ; i++)
         {
             // Check again in case "-h" isn't the first option.
-            if (strcmp(argv[i], "-h") == 0 || strcmp(argv[i], "-help") == 0)
+            if (strcmp(argv[i], "-h") == 0 || strcmp(argv[i], "-help") == 0 || strcmp(argv[i], "-help") == 0))

```

Comment:
There is an extra trailing parenthesis here too. 

Once help mode is triggered we should immediately emit the help message and then exit. You could implement that either with a `goto` or by moving the help message into its own function.

Also, what happens if I run `./capture` with no arguments?

Reply:
I fixed the bug.  Moving the help functionality may be handled when the code is refactored.